code/modules/vehicles/cargo_train.dm

```diff
@@ -34,12 +33,16 @@
 	load_item_visible = 1
 	load_offset_x = 0
 	load_offset_y = 4
-	buckle_pixel_shift = @"{'x':0,'y':0,'z':8}"
+
+/obj/vehicle/train/cargo/trolley/Initialize()
+	buckle_pixel_shift = list("x" = 0, "y" = 0, "z" = 8)
```

Comment:
why do this in init instead of var declaration?

Reply:
List declarations at compile time create an invisible magic `(init)` proc.

Reply:
Would be fine, just realized these ones aren't directional so are nominally constant for the compiler's sake.